### DISCUSSION TITLE
Calling new Queue() twice with the same queue name

### DISCUSSION THREAD

**Discussion by CharlesAntoineCaron** [taskforcesh/bullmq#1172] [Q&A]
With the following code:

`const myQueue1 = new Queue<ISomeInterface>("MY QUEUE", {...});`
`const myQueue2 = new Queue<ISomeInterface>("MY QUEUE", {...});`

Are myQueue1 and myQueue2 the same queue? Or does it create 2 different queues?

Thanks.

**Comment by manast** [comment]
Two instances of the same queue...
